### ISSUE TITLE
Meeting page template improvements

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#2108]
**What we're after:**
 In doing the hearing pages for the upcoming June 27-28 rulemaking hearing ([Day 1](https://www.fec.gov/updates/june-27-2018-public-hearing-day-1/)) ([Day 2](https://www.fec.gov/updates/june-28-2018-public-hearing-day-2/)), we discovered some things about the meeting page template that need improvement:
- We're unable to make a single page that works for a hearing lasting more than one day. Even if you enter a second day, it defaults to the first day.
- When we select type of meeting, the only options are "open meeting" or "executive session" - there's no option for "hearing"
- We need a place to enter optional introductory text for hearings (not really something needed for open meetings as that text is hard-coded into the template).
- Any page for a hearing would need to have all the usual audio/document options now on the template, plus see #1323, #2095 and #2099 for additional improvements on these.

Completion criteria:
- [ ] Update template to allow for meetings that last more than one day, so that only one page needs to be created per hearing/meeting 
- [ ] Add optional block for optional intro text.
- [ ] Implement any change or improvement in audio/document options on any changed template or new template for hearings.

**Comment by AmyKort** [comment]
Thanks @dorothyyeager and @llienfec !

**Comment by dorothyyeager** [comment]
We were able to come up with a workaround, but it's not ideal due to the meeting date default to one day.

See https://www.fec.gov/updates/june-27-28-2018-public-hearing/. 

Without the ability to do embedded video or offer additional intro text, a second page is under development for that hearing to contain that information. Ideally though, by the next time we've got this situation, it would be good to have a hearing template in place and modifications made to the meeting page template.
